docs/deprecated.md

```diff
@@ -50,6 +50,7 @@ The table below provides an overview of the current status of deprecated feature
 
 Status     | Feature                                                                                                                            | Deprecated | Remove
 -----------|------------------------------------------------------------------------------------------------------------------------------------|------------|------------
+Deprecated | [Support for the legacy `~/.dockercfg` configuration file for authentication](#support-for-legacy-dockercfg-configuration-files)   | v20.03.0   | -
```

Comment:
Isn't it removed instead of deprecated ?

Reply:
It's only "deprecated" for now, as https://github.com/docker/cli/pull/2504 isn't merged yet (and won't be for the 20.0x release)